### PR TITLE
Improve logging 

### PR DESCRIPTION
**Pull request by sdelamo**
ApplicationLoadBalancerRequestEvent and APIGatewayV2HTTPEvent uses Lombok @Data which applies toString https://projectlombok.org/features/Data

APIGatewayProxyRequestEvent overrides toString to render a JSON representation of the object

### CODE REVIEW

**Review by timyates — CHANGES_REQUESTED** [review]
Should we do `ApplicationLoadBalancerHandler` as well for ALB events?

`ApplicationLoadBalancerRequestEvent` also uses Lombok

Also, it's not json, it's just stuck between `{` and `}` for some reason 😉

**Comment by sdelamo** [comment]
> Should we do `ApplicationLoadBalancerHandler` as well for ALB events?
> 
> `ApplicationLoadBalancerRequestEvent` also uses Lombok
> 
> Also, it's not json, it's just stuck between `{` and `}` for some reason 😉

I am logging in `ApplicationLoadBalancerHandler` as well. Is that what you meant?

File: function-aws-api-proxy/src/main/java/io/micronaut/function/aws/proxy/alb/ApplicationLoadBalancerHandler.java [hunk 1]
```diff
@@ -40,6 +44,9 @@ protected ServletExchange<ApplicationLoadBalancerRequestEvent, ApplicationLoadBa
         ApplicationLoadBalancerRequestEvent request,
         ApplicationLoadBalancerResponseEvent response
     ) {
+        if (LOG.isTraceEnabled()) {
+            LOG.info("Request: {}", request);
```
Comment (sdelamo) [inline function-aws-api-proxy/src/main/java/io/micronaut/function/aws/proxy/alb/ApplicationLoadBalancerHandler.java:48]:
```suggestion
            LOG.trace("Request: {}", request);
```

File: function-aws-api-proxy/src/main/java/io/micronaut/function/aws/proxy/payload2/APIGatewayV2HTTPEventHandler.java [hunk 2]
```diff
@@ -45,6 +48,9 @@ protected ServletExchange<APIGatewayV2HTTPEvent, APIGatewayV2HTTPResponse> creat
         APIGatewayV2HTTPEvent request,
         APIGatewayV2HTTPResponse response
     ) {
+        if (LOG.isTraceEnabled()) {
+            LOG.info("Request: {}", request);
```
Comment (sdelamo) [inline function-aws-api-proxy/src/main/java/io/micronaut/function/aws/proxy/payload2/APIGatewayV2HTTPEventHandler.java:52]:
```suggestion
            LOG.trace("Request: {}", request);
```

File: function-aws-api-proxy/src/main/java/io/micronaut/function/aws/proxy/payload1/ApiGatewayProxyEventHandler.java [hunk 3]
```diff
@@ -45,6 +48,9 @@ protected ServletExchange<APIGatewayProxyRequestEvent, APIGatewayProxyResponseEv
         APIGatewayProxyRequestEvent request,
         APIGatewayProxyResponseEvent response
     ) {
+        if (LOG.isTraceEnabled()) {
+            LOG.info("Request: {}", request);
```
Comment (sdelamo) [inline function-aws-api-proxy/src/main/java/io/micronaut/function/aws/proxy/payload1/ApiGatewayProxyEventHandler.java:52]:
```suggestion
            LOG.trace("Request: {}", request);
```

**Review by timyates — APPROVED** [review]

**Pull request merged**